<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoHDR_MainWindow</class>
 <widget class="QMainWindow" name="AutoHDR_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>967</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>967</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoHDR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLowerCriteria">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max white px %</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUpperCriteria">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max black px %</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNbImgMax">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max nb images</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderNbImgMax">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>1</height>
     </size>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderUpperCriteria">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>1</height>
     </size>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderLowerCriteria">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>1</height>
     </size>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtUpperCriteria">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLowerCriteria">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtNbImgMax">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture parameters</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxISO">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="boxAperture">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="boxMaxExposure">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labeISO">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ISO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAperture">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aperture</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxExposure">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max exposure</string>
    </property>
   </widget>
   <widget class="LiveViewDisplay" name="liveViewDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>500</width>
      <height>333</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="boxExposure">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelExposure">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start exposure</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open...</string>
     </property>
     <addaction name="actionConfiguration_load"/>
     <addaction name="actionSequence_load"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save...</string>
     </property>
     <addaction name="actionSequence_save"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionConfiguration_load">
   <property name="text">
    <string>Configuration file...</string>
   </property>
  </action>
  <action name="actionSequence_load">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sequence file...</string>
   </property>
  </action>
  <action name="actionSequence_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sequence file...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LiveViewDisplay</class>
   <extends>QWidget</extends>
   <header>liveview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
